src/interfaces/ISablierLockup.sol

```diff
@@ -11,15 +11,15 @@ interface ISablierLockup is ISablierLockupBase {
                                        EVENTS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @notice Emitted when a stream is created using LD model.
+    /// @notice Emitted when a LD stream is created.
```

Comment:
```suggestion 
    /// @notice Emitted when an LD stream is created.
```

Same with LL and LT below.

Reply:
I've thought about this when I authored the commit, and I decided to stick with `a LD` for the following reasons:

1. Easier to make refactors
2. The text can be read as "A Lockup Dynamic" instead of the acronyms